test/fixtures/v4/did-signed-wrapped.json

```diff
@@ -0,0 +1,51 @@
+{
+  "@context": [
+    "https://www.w3.org/2018/credentials/v1",
+    "https://schemata.openattestation.com/com/openattestation/4.0/alpha-context.json"
+  ],
+  "type": ["VerifiableCredential", "OpenAttestationCredential"],
+  "issuanceDate": "2021-03-08T12:00:00+08:00",
+  "validFrom": "2021-03-08T12:00:00+08:00",
+  "name": "Republic of Singapore Driving Licence",
+  "issuer": {
+    "id": "did:ethr:0xB26B4941941C51a4885E5B7D3A1B861E54405f90",
+    "type": "OpenAttestationIssuer",
+    "name": "Government Technology Agency of Singapore (GovTech)",
+    "identityProof": { "identityProofType": "DNS-DID", "identifier": "example.openattestation.com" }
+  },
+  "credentialStatus": { "type": "OpenAttestationCredentialStatus", "credentialStatusType": "NONE" },
+  "renderMethod": {
+    "type": "OpenAttestationRenderMethod",
+    "renderMethodType": "EMBEDDED_RENDERER",
+    "name": "GOVTECH_DEMO",
+    "url": "https://demo-renderer.openattestation.com"
+  },
+  "credentialSubject": {
+    "id": "urn:uuid:a013fb9d-bb03-4056-b696-05575eceaf42",
+    "type": ["DriversLicense"],
```

Comment:
just curious, how did you get this value `urn:uuid:a013fb9d-bb03-4056-b696-05575eceaf42`?

Reply:
This was just a randomly generated UUID. Can be any form of unique identifier